runtime/src/lib.rs

```diff
@@ -369,7 +369,7 @@ impl_opaque_keys! {
 
 impl pallet_transaction_payment::Config for Runtime {
     type RuntimeEvent = RuntimeEvent;
-    type OnChargeTransaction = CurrencyAdapter<Balances, ()>;
+    type OnChargeTransaction = FungibleAdapter<Balances, ()>;
```

Comment:
Main differences w.r.t. Currency Adapter ?

Reply:
CurrencyAdapter Implements transaction payment for a pallet implementing the Currency trait whereas FungibleAdapter Implements transaction payment for a pallet implementing the  frame_support::traits::fungible trait.

So really the difference is the deprecation of the Currency trait in favor of the Fungible trait, why?

The reason on this change seems to be because of an issue on balance calculation existing on CurrencyAdapter, specified here [](https://github.com/paritytech/polkadot-sdk/issues/1833)